woocommerce/woocommerce-android
# [Deposit Summary] Add payments hub deposit summary currency selected event tracking

Closes: #10189


### Description

The PR adds tracking of the `PAYMENTS_HUB_DEPOSIT_SUMMARY_CURRENCY_SELECTED` event with currency property, only when it was selected by a user

```
Tracked: payments_hub_deposit_summary_currency_selected, Properties: {"currency":"eur","blog_id":222936016,"is_wpcom_store":false,"was_ecommerce_trial":false,"plan_product_slug":"jetpack_security_daily","is_debug":true,"site_url":"https:\/\/jhmulticurrency.mystagingwebsite.com"}
```

### Testing instructions

* Open more -> payments on a site that has more than 1 currency
* Select a currency
* Notice `PAYMENTS_HUB_DEPOSIT_SUMMARY_CURRENCY_SELECTED` is tracked
